FEAT: added initial support and styling for a zigzag cards layout and ID badge moment type

vsealsnutt opened the pull request.

vsealsnutt committed:
FEAT: added initial support and styling for a zigzag cards layout and ID badge moment type

emulatingkat reviewed (approved):
Thank you for this work!

kennethsn committed:
Merge remote-tracking branch 'origin/feat/169_id-badge-moment' into feat/169_id-badge-ksn

kennethsn committed:
DOCS: release dev version 1.0.0-dev-50

kennethsn committed:
FEAT: add templating logic for moment, story, and collection fields

also refactors id badge and awards moments

BREAKING CHANGE: - award moment -> awards moment
#169

vsealsnutt committed:
Merge pull request #189 from kennethsn/feat/169_id-badge-ksn

FEAT: Add templating support for collection, story, and moment fields
